test/DynamoCoreWpfTests/PreferencesGeometryScalingTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Windows;
+using System.Windows.Controls.Primitives;
+using Dynamo.Graph.Nodes;
+using Dynamo.Tests;
+using Dynamo.ViewModels;
+using Dynamo.Wpf.Views;
+using DynamoCoreWpfTests.Utility;
+using NUnit.Framework;
+
+namespace DynamoCoreWpfTests
+{
+    [TestFixture]
+    class PreferencesGeometryScalingTests : DynamoTestUIBase
+    {
+        public override void Open(string path)
+        {
+            base.Open(path);
+            DispatcherUtil.DoEvents();
+        }
+
+        protected override void GetLibrariesToPreload(List<string> libraries)
+        {
+            libraries.Add("VMDataBridge.dll");
+            libraries.Add("ProtoGeometry.dll");
+            libraries.Add("DesignScriptBuiltin.dll");
+            base.GetLibrariesToPreload(libraries);
+        }
+
+        /// <summary>
+        /// The test validates that when RunType=Automatic and the MarkNodesAsModifiedAndRequestRun() method is called the graph execution is kicked off
+        /// The test validates that when RunType=Manual and the MarkNodesAsModifiedAndRequestRun() method is called the graph execution is NOT kicked off
+        /// </summary>
+        [Test]
+        [TestCase("Automatic", ElementState.Warning)]
+        [TestCase("Manual", ElementState.Active)]
```

Comment:
Nice, but do you think we can these two string from DynamoCoreWPF resources just so in the future if we change those the test still pass?

Reply:
@QilongTang I had to split the tests in two (one for Automatic and other for Manual) due that the Resources strings can not be used as a parameters in nUnit Tests parameters (since parameters expected should be constant values).
https://github.com/DynamoDS/Dynamo/pull/11867/commits/699036291fb12bdd05c06883b1ccf67a5f9d6b2e